src/components/Grid/Grid.tsx

```diff
@@ -346,17 +282,13 @@ export const Grid: FC<GridProps> = ({
         }
 
         setClassicArrowStartId(null);
```

Comment:
Can we remove "classic" from all variable/function names?

Reply:
jepp, i egen pr